docs/Tutorials/localnet/README.md

```diff
@@ -1,19 +1,23 @@
 ---
 Title: Localnet
-Description: A tutorial of how to build `budgetd` and bootstrap local network.
+Description: How to build the `budgetd` binary and bootstrap a local network.
 ---
 
-### Get Budget Module source code
+# Localnet 
+
+This tutorial provides steps to build the `budgetd` binary and bootstrap a local network.
+
+# Get Budget Module Source Code
 
 ```bash
 git clone https://github.com/tendermint/budget.git
 cd budget
 make install
 ```
 
-### Boostrap
+### Bootstrap a Chain 
 
-The following script is prepared to bootstrap a single chain with a single validator in your local machine. Copy the script and run them in your terminal.
+The following script bootstraps a single chain in your local machine with a single validator. Copy the script and run it in your terminal.
```

Comment:
do we copy the code into a file with an *.sh file extension? or can this code be pasted into a terminal window and run in entirety?

Reply:
seems like we can provide a script file in the codebase that the user can download?

Reply:
It is meant to let developers copy the codes and paste them into a terminal window to run; however, they can also create a file ending with an *.sh and copy and paste those codes (it requires them to hard code `#!/bin/bash` as starting line). It is actually a great idea to prove a script file somewhere so that they can just download it. Let's create an issue for that and handle it in a separate PR.